site/docs/v1/tech/apis/system.adoc

```diff
@@ -108,46 +108,35 @@ link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas
 [.endpoint]
 .URI
 --
-[method]#GET# [uri]#/api/system/log/export``?dateTimeSecondsFormat=\{dateTimeSecondsFormat\}&lastNBytes=\{lastNBytes\}&zoneId=\{zoneId\}``#
+[method]#GET# [uri]#/api/system/log/export``?dateTimeSecondsFormat=\{dateTimeSecondsFormat\}&includeArchived=true&lastNBytes=\{lastNBytes\}&zoneId=\{zoneId\}``#
```

Comment:
Would you ever have both `includeArchived` and `lastNBytes` in the same request? If not, I'd pick one. 

We don't have to have every parameter displayed on the url.

Another option would be to have two different `[.endpoint]` sections, one with each type of call. That is useful if both are expected to be frequently called. Otherwise I'd pick the one you feel is more useful. (Probably `includeArchived`, but that's just a guess.)